services/gateway/apis/http-version3/swagger/definitions/schemas.json

```
@@ -203,6 +203,299 @@
             }
         }
     },
+    "headerSchema": {
```

The swagger definition identifiers must always be defined in `PascalCase`. Please ensure to update the existing ones as and when we come across them.
```suggestion
    "HeaderSchema": {
```